Gruntfile.js

```diff
@@ -1,6 +1,63 @@
 'use strict';
 module.exports = function(grunt) {
+    const parsedIconPicker = 'prod/src/js/iconpicker.js';
+    const tempIconsFile = '.icons.temp';
     grunt.initConfig({
+        download: {
+            somefile: {
+                src: ['https://raw.githubusercontent.com/FortAwesome/Font-Awesome/master/advanced-options/metadata/icons.yml'],
```

Comment:
Please use 5.0.8 instead of master, which can lead to problems if the file ever changes its format or disappears there.

`https://raw.githubusercontent.com/FortAwesome/Font-Awesome/5.0.8/advanced-options/metadata/icons.yml`